dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/internal/data/upload/UploadStatusTest.kt

```diff
@@ -99,15 +97,19 @@ internal class UploadStatusTest {
         status.logStatus(
             fakeContext,
             fakeByteSize,
-            mockLogger
+            mockLogger,
+            requestAttempts = fakeRequestAttempts
         )
 
         // Then
         mockLogger.verifyLog(
             InternalLogger.Level.WARN,
             listOf(InternalLogger.Target.USER),
             "Batch [$fakeByteSize bytes] ($fakeContext) failed " +
-                "because of a network error (${status.throwable!!.message}); we will retry later."
+                "because of a network error (${status.throwable!!.message}); we will retry later." + " " +
+                UploadStatus.RETRY_REQUEST_LOG_MESSAGE_FORMAT.format(
+                    Locale.US, fakeRequestAttempts, status.code
+                )
```

Comment:
> [!WARNING]
> You should not do that as you're using the same algorithm to create the expected output as the production code.

Reply:
yup I realized that, I am fixing it now, I am afraid I will have to do this in function of the targe.